src/util/animate.js

```diff
@@ -11,22 +71,27 @@
   /**
    * Changes value from one to another within certain period of time, invoking callbacks as value is being changed.
    * @memberOf fabric.util
-   * @param {Object} [options] Animation options
-   * @param {Function} [options.onChange] Callback; invoked on every value change
-   * @param {Function} [options.onComplete] Callback; invoked when value change is completed
-   * @param {Number} [options.startValue=0] Starting value
-   * @param {Number} [options.endValue=100] Ending value
-   * @param {Number} [options.byValue=100] Value to modify the property by
-   * @param {Function} [options.easing] Easing function
-   * @param {Number} [options.duration=500] Duration of change (in ms)
-   * @param {Function} [options.abort] Additional function with logic. If returns true, onComplete is called.
-   * @returns {Function} abort function
+   * @param {AnimationOptions} [options] Animation options
```

Comment:
this helps with tpyescript, i m not sure if jsdocs still sees it.

Reply:
I see in VSCode... not sure as well.
What's the decision?

Reply:
456a36052dda7a16009849017a8122f504e81b00

Reply:
test if docs build is enough. i can do that don't worry about it now

Reply:
OK.
So, this is stable and can be merged